doc/nrf/app_boards.rst

```diff
@@ -58,15 +78,20 @@ Also see the :ref:`zephyr:boards` section in the Zephyr documentation.
 |                   |            |                                                                 |                                       |
 |                   |            |                                                                 | ``thingy53_nrf5340_cpuapp``           |
 |                   |            |                                                                 |                                       |
-|                   |            |                                                                 | ``thingy53_nrf5340_cpuapp_ns``        |
+|                   |            |                                                                 | ``thingy53_nrf5340_cpuapp_ns``*       |
```

Comment:
The build does not seem to like the asterisk at this spot. Would it work adding a backslash just before the asterisk, like this \*?
Or enclosing it in double back-quotes (inline literals), like this ``*``.

Do we need this though? It feels a bit like a footnote :)